src/components/PortfolioProjectDialog/PortfolioProjectDialogDesktop.tsx

```diff
@@ -43,11 +62,18 @@ const useStyles = makeStyles((theme) => ({
 
 export const PortfolioProjectDialogDesktop = (props: PortfolioProjectDialogVariantProps) => {
   const classes = useStyles();
+  const [dialogScroll, setDialogScroll] = useState(0);
+
+  const controlsTransform = `translateY(${Math.max(dialogScroll - 16, 0)}px)`;
```

Comment:
What is this 16 number, is it some margin on padding?

Reply:
It's a space between the dialog and its controls